### ISSUE TITLE
Blurry results and non-recoverable facial features in CelebA-HQ dataset

### ISSUE THREAD

**Issue by SherryXTChen** [NVIDIA/partialconv#34]
Hi. Has anyone tried to reproduce the painting results on CelebA-HQ dataset? This model is trained on a randomly select subset of CelebA-HQ of size 27k for 3 days on a GPU and fine-tuned for half of a day using batch size 6. And here are some outputs I get, where the inputs come from my screenshot of the paper:
![my_result](https://user-images.githubusercontent.com/32473234/116328247-758f2c80-a77d-11eb-8cbb-5a8ed1cb0db0.png)

For comparison, here are the results shown in the paper (Fig 8):
<img width="643" alt="sample" src="https://user-images.githubusercontent.com/32473234/116328293-8fc90a80-a77d-11eb-90ce-88deb727fcec.png">

For my code, the inpainted regions are blurry when the masked areas are big and the masked facial features are not recovered. I wonder if anyone has encountered similar issues or has any guess on what might be the reason. Thanks!

**Comment by SURABHI-GUPTA** [comment]
@SherryXTChen How did you train the inpainting model?

**Comment by SherryXTChen** [comment]
@SURABHI-GUPTA I was using my own perceptual loss code and there was a bug. After fixing it, I was able to follow the paper and get compatible results. Thanks.

**Comment by shangdibufashi** [comment]
@SherryXTChen Great job! Would please share your code and working model that trained using CelebA-HQ?  Sadly, I don't have powerful GPU device...
